.travis.yml

```diff
@@ -136,6 +136,10 @@ matrix:
     - ARROW_TRAVIS_GANDIVA_JAVA=1
     - ARROW_TRAVIS_OPTIONAL_INSTALL=1
     - ARROW_BUILD_WARNING_LEVEL=CHECKIN
+    # ARROW-3803: The Xcode 8.3 image has Boost libraries in /usr/local/lib
+    # which can get loaded before the toolchain Boost libraries. May be
+    # possible to fix without static linking
```

Comment:
`LD_LIBRARY_PATH` is less invasive.

Reply:
LD_LIBRARY_PATH is already set... 

https://github.com/apache/arrow/blob/master/ci/travis_env_common.sh#L63